Description: Upstream changes introduced in version 11.86-3
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 auctex (11.86-3) unstable; urgency=low
 .
   * Port for Nexenta 4
 .
 The person named in the Author field signed this changelog entry.
Author: Peter Zaikin <peter.zaikin@nexenta.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ auctex-11.86/config.log
@@ -0,0 +1,183 @@
+This file contains any messages produced by compilers while
+running configure, to aid debugging if configure makes a mistake.
+
+It was created by auctex configure 11.86, which was
+generated by GNU Autoconf 2.65.  Invocation command line was
+
+  $ ./configure --prefix=/usr/ --infodir=/usr/share/info/ --with-emacs=emacs
+
+## --------- ##
+## Platform. ##
+## --------- ##
+
+hostname = nexenta10
+uname -m = i86pc
+uname -r = 5.11
+uname -s = SunOS
+uname -v = testws
+
+/usr/bin/uname -p = i386
+/bin/uname -X     = System = SunOS
+Node = nexenta10
+Release = 5.11
+KernelID = testws
+Machine = i86pc
+BusType = <unknown>
+Serial = <unknown>
+Users = <unknown>
+OEM# = 0
+Origin# = 1
+NumCPU = 1
+
+/bin/arch              = i86pc
+/usr/bin/arch -k       = i86pc
+/usr/convex/getsysinfo = unknown
+/usr/bin/hostinfo      = unknown
+/bin/machine           = unknown
+/usr/bin/oslevel       = unknown
+/bin/universe          = unknown
+
+PATH: /usr/gnu/bin
+PATH: /usr/bin
+PATH: /sbin
+PATH: /usr/sbin
+PATH: /usr/local/bin
+PATH: /usr/local/sbin
+
+
+## ----------- ##
+## Core tests. ##
+## ----------- ##
+
+configure:1731: checking for make
+configure:1747: found /usr/bin/make
+configure:1758: result: make
+configure:1774: checking whether make sets $(MAKE)
+configure:1796: result: yes
+configure:1842: checking for a BSD-compatible install
+configure:1910: result: /usr/gnu/bin/install -c
+configure:1923: checking for date in ChangeLog
+configure:1930: result: 2010-02-21
+configure:1932: checking for release in ChangeLog
+configure:1941: result: 11.86
+configure:1992: checking for emacs
+configure:2025: result: no
+configure:1992: checking for emacs
+configure:2025: result: no
+configure:1992: checking for xemacs
+configure:2025: result: no
+configure:2035: error: (X)Emacs not found!  Aborting!
+
+## ---------------- ##
+## Cache variables. ##
+## ---------------- ##
+
+ac_cv_env_INSTALL_INFO_set=''
+ac_cv_env_INSTALL_INFO_value=''
+ac_cv_env_MAKEINFO_set=''
+ac_cv_env_MAKEINFO_value=''
+ac_cv_env_PERL_set=''
+ac_cv_env_PERL_value=''
+ac_cv_env_build_alias_set=''
+ac_cv_env_build_alias_value=''
+ac_cv_env_host_alias_set=''
+ac_cv_env_host_alias_value=''
+ac_cv_env_target_alias_set=''
+ac_cv_env_target_alias_value=''
+ac_cv_path_install='/usr/gnu/bin/install -c'
+ac_cv_prog_MAKECMD=make
+ac_cv_prog_make_make_set=yes
+
+## ----------------- ##
+## Output variables. ##
+## ----------------- ##
+
+AUCTEXDATE='2010-02-21'
+AUCTEXVERSION='11.86'
+DEFS=''
+DVIPS=''
+ECHO_C='\c'
+ECHO_N=''
+ECHO_T=''
+EMACS=''
+EMACS_FLAVOR=''
+HAVE_mule=''
+INSTALL_DATA='${INSTALL} -m 644'
+INSTALL_INFO=''
+INSTALL_PROGRAM='${INSTALL}'
+INSTALL_SCRIPT='${INSTALL}'
+LIBOBJS=''
+LIBS=''
+LTLIBOBJS=''
+MAKECMD='make'
+MAKEINFO=''
+MULEELC=''
+MULESRC=''
+PACKAGE_BUGREPORT='bug-auctex@gnu.org'
+PACKAGE_NAME='auctex'
+PACKAGE_STRING='auctex 11.86'
+PACKAGE_TARNAME='auctex'
+PACKAGE_URL=''
+PACKAGE_VERSION='11.86'
+PATH_SEPARATOR=':'
+PDFTEX=''
+PERL=''
+SET_MAKE=''
+SHELL='/bin/sh'
+TEX=''
+TEXI2DVI=''
+TEXI2HTML=''
+TEXI2PDF=''
+XEMACS=''
+auctexstartfile=''
+autodir=''
+bindir='${exec_prefix}/bin'
+build_alias=''
+datadir='${datarootdir}'
+datarootdir='${prefix}/share'
+docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
+dvidir='${docdir}'
+exec_prefix='NONE'
+host_alias=''
+htmldir='${docdir}'
+includedir='${prefix}/include'
+infodir='/usr/share/info'
+libdir='${exec_prefix}/lib'
+libexecdir='${exec_prefix}/libexec'
+lispautodir=''
+lispdir=''
+lisppackagedatadir=''
+lisppackagelispdir=''
+lisptexsite=''
+localedir='${datarootdir}/locale'
+localstatedir='${prefix}/var'
+mandir='${datarootdir}/man'
+oldincludedir='/usr/include'
+packagedatadir=''
+packagedir=''
+packagelispdir=''
+pdfdir='${docdir}'
+prefix='/usr'
+preview_enabled=''
+program_transform_name='s,x,x,'
+psdir='${docdir}'
+sbindir='${exec_prefix}/sbin'
+sharedstatedir='${prefix}/com'
+subdirs=''
+sysconfdir='${prefix}/etc'
+target_alias=''
+texsite=''
+
+## ----------- ##
+## confdefs.h. ##
+## ----------- ##
+
+/* confdefs.h */
+#define PACKAGE_NAME "auctex"
+#define PACKAGE_TARNAME "auctex"
+#define PACKAGE_VERSION "11.86"
+#define PACKAGE_STRING "auctex 11.86"
+#define PACKAGE_BUGREPORT "bug-auctex@gnu.org"
+#define PACKAGE_URL ""
+
+configure: exit 1
